src/modules/commander/commander.cpp

```diff
@@ -1158,7 +1158,7 @@ int commander_thread_main(int argc, char *argv[])
 #endif
 
 	if (argc > 2) {
-		if (!strcmp(argv[3],"-hil")) {
+		if (!strcmp(argv[2],"-hil")) {
```

Comment:
I need to re-check this on Pixhawk - if you need this to go in fast we can ifdef this to pull it in.